src/dev/run_i18n_extract.ts

```diff
@@ -35,7 +35,7 @@ import { resolve } from 'path';
 import { createFailError, run } from '@osd/dev-utils';
 import { ErrorReporter, serializeToJson, serializeToJson5, writeFileAsync } from './i18n';
 import { extractDefaultMessages, mergeConfigs } from './i18n/tasks';
-
+import { ListrContext } from './run_i18n_check';
```

Comment:
Nit: This can come from a common source instead of another file.

Reply:
./i18n/tasks or ./i18n. Right?